Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -351,7 +351,7 @@ def self.changed_packages(tap, original_commit)
         nil
       end
     end.compact
-    formulae + casks
+    T.must(formulae).compact + casks
```

Comment:
There's an asymmetry here bc the `map` in the `formulae` assignment contains a `break`, which results in a `nil` return value. (Not something I knew, and a pretty impressive bit of static analysis IMO)